### PR TITLE
Add path and argparse to the README

### PR DESCRIPTION
**Pull request by pepe**
I just copied old readmes for both projects into a spork one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Various Janet utility modules. Spork aims to be grab bag of useful Janet functionality that
 does not belong in the core library.
 
+To use all features mentioned bellow in ones program you can do `(use spork)` to
```
Comment (sogaiu) [inline README.md:6]: s/bellow/below/ and s/ones/one's/ ?

File: README.md [hunk 2]
```diff
@@ -93,6 +93,91 @@ A simple remote procedure call tool for Janet.
...
 ```
 
+## Path
+
+Simple path manipulation module for Janet. Supports manipulation both
+windows and posix paths on any platform, and provides functions that
+work according to the current host platform.
+
+All functions have three forms, under `path`, `path/win`, and `path/posix`. The
+prefix
```
Comment (sogaiu) [inline README.md:107]: s/prefix/suffix/ ?

File: README.md [hunk 3]
```diff
@@ -93,6 +93,91 @@ A simple remote procedure call tool for Janet.
...
+indicates which type of path the function manipulates.
+
+```clojure
+(import spork/path)
+
+# Examples for a non-windows system, use path/win/ for windows and
```
Comment (sogaiu) [inline README.md:113]: May be using backticks are `path/win/` is good?

File: README.md [hunk 4]
```diff
@@ -93,6 +93,91 @@ A simple remote procedure call tool for Janet.
...
+# path/posix/ for posix.
```
Comment (sogaiu) [inline README.md:114]: May be using backticks around `path/posix/` is good?

File: README.md (hunk 2 above)
Comment (pepe) [reply README.md:107]: These are all three prefixes.

File: README.md (hunk 3 above)
Comment (pepe) [reply README.md:113]: They are in the code already.

File: README.md (hunk 4 above)
Comment (pepe) [reply README.md:114]: They are in the code already.

File: README.md (hunk 1 above)
Comment (pepe) [reply README.md:6]: This is the sentence I added myself :facepalm:

File: README.md (hunk 2 above)
Comment (sogaiu) [reply README.md:107]: Oh I see.  My bad :)

File: README.md (hunk 3 above)
Comment (sogaiu) [reply README.md:113]: Missed that -- sorry :)

File: README.md (hunk 4 above)
Comment (sogaiu) [reply README.md:114]: Oops :)

**Comment by sogaiu** [comment]
LGTM

**Pull request merged**